src/client/Microsoft.Identity.Client/AuthScheme/PoP/InMemoryCryptoProvider.cs

```diff
@@ -36,7 +44,10 @@ private void InitializeSigningKey()
 
         public byte[] Sign(byte[] payload)
         {
-            return _signingKey.SignData(payload, HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1);
+            return _signingKey.SignData(
+                payload,
+                HashAlgorithmName.SHA256,
+                RSASignaturePadding.Pss);
```

Comment:
will this affect generic authority? i.e. if the authority does not support PSS padding

Reply:
POP support for non-AAD authorities is not guarateed. POP SHR API is anyway experimental.